experiment/evaluator.py

```diff
@@ -259,8 +260,9 @@ def _parse_fuzz_cov_info_from_libfuzzer_logs(
     return initcov, donecov, lastround
 
   def _stack_func_is_of_testing_project(self, stack_frame: str) -> bool:
-    return bool(CRASH_STACK_WITH_SOURCE_INFO.match(stack_frame)) and (
-        LLVM_SOURCE_PATH_PREFIX not in stack_frame)
+    return (bool(CRASH_STACK_WITH_SOURCE_INFO.match(stack_frame)) and
+                 LIBFUZZER_LOG_STACK_FRAME_LLVM not in stack_frame and
+                 LIBFUZZER_LOG_STACK_FRAME_CPP not in stack_frame)
```

Comment:
ops, indentation is incorrect. Check the linter failure: )
Once this is fixed, we can merge this and run another exp in the parent PR.